armi/physics/neutronics/parameters.py

```diff
@@ -579,6 +579,7 @@ def linPowByPinGamma(self, value):
     with pDefs.createBuilder(
         default=0.0,
         location=ParamLocation.AVERAGE,
+        categories=[parameters.Category.detailedAxialExpansion],
```

Comment:
Is this related to this PR, or just a bonus?

Reply:
It's part of this PR. Some of these reaction rate parameters need to be mapped when performing gamma transport using a downstream plugin.

Before this PR, the uniform mesh converter would `deepcopy` blocks from the source assembly to the destination assembly in `makeAssemWithUniformMesh`, and these parameters would have values acquired from the `deepcopy`. Now that we have replaced the `deepcopy` with a less comprehensive copy function, the parameters need to be explicitly mapped so that the values are present on the uniform mesh reactor.

Reply:
Yes, I think we really only need `rateFis` and `rateProdFis` for the particular use case that I'm thinking of. But there could be other cases where other reaction rates are desired. I didn't know exactly where to draw the line so I just included all of them instead of being picky. But if we're really trying to drill down on performance then we could limit it to just those fission rate parameters.